unittests/test-contracts/code_version_test/code_version_test.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <eosio/eosio.hpp>
+
+namespace eosio {
```

Comment:
It's better not to put this stuff in namespace eosio, as that causes a headache when cdt is eventually updated.